recipes/pytensor/meta.yaml

```
@@ -0,0 +1,109 @@
+{% set version = "2.8.10" %}
+
+package:
+  name: pytensor-suite  # https://github.com/conda/conda-build/issues/3933
```

Just a suggestion, not a requirement to merge this but I'd rather have comments on top and leave the inline `#` for selectors.

```suggestion
  # https://github.com/conda/conda-build/issues/3933
  name: pytensor-suite 
```